exec/src/klio_exec/commands/run.py

```diff
@@ -287,11 +284,8 @@ def _verify_packaging(self):
                     "Use with caution."
                 )
 
-            if not any([fnapi_enabled, setup_file_exists, reqs_file_exists]):
-                logging.error(
-                    "setup.py and/or requirements file either "
-                    "unspecified or not found."
-                )
+            if not any([fnapi_enabled, setup_file_exists]):
+                logging.error("setup.py file either unspecified or not found.")
```

Comment:
What do you think of adding a line, something like:

```py
if not any([fnapi_enabled, setup_file_exists]):
  if has_reqs_file:
    logging.error("Klio does not support requirements.txt file. Please use blablahbla")
  else:
    logging.error("setup.py file either unspecified or not found.")
  ...
```

Reply:
Makes sense! Added.